bindings/cs/rl.net.native/binding_sender.cc

```diff
@@ -0,0 +1,27 @@
+#include "binding_sender.h"
+#include <climits>
+
+using namespace reinforcement_learning;
+
+namespace rl_net_native {
+  int binding_sender::init(api_status* status)
+  {
+    return this->vtable.init(managed_handle, status);
+  }
+
+  int binding_sender::v_send(const buffer& data, api_status* status)
+  {
+    size_t length = data->buffer_filled_size();
```

Comment:
`auto length`

Reply:
I prefer to specify the types explicitly - it makes it much easier to read for people who are not using tools beyond notepad or who have not seen the code recently, and do not know the actual types. Using auto also tends to lead people to ignore auto-coersion between type-sizes, which we keep encountering in the form of precision-loss warnings.

While I will not comment about others' use of it, I view `auto` (and `var` in C#) as harmful in the majority of cases, particularly when the type name is short.